Node v19 throws an error: 'The "name" argument must be specified'

### Environment

Nuxi 3.2.3                                                                                                               
                                                                                                                                      
RootDir: /app                                                                                                                       
Nuxt project info:                                                                                                                   

------------------------------
- Operating System: `Linux`
- Node Version:     `v19.7.0`
- Nuxt Version:     `3.2.3`
- Nitro Version:    `2.3.1`
- Package Manager:  `pnpm@7.29.1`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

docker run --rm -it node:19-alpine sh
>  npm add -g pnpm
> pnpm dlx nuxi init app
> cd app
>  pnpm install

### Describe the bug

Nuxi 3.3.1                                                                                                                            

 ERROR  The "name" argument must be specified                                                                                         

  at new NodeError (node:internal/errors:399:5)
  at Performance.mark (node:internal/perf/performance:125:13)
  at normalizedModule (node_modules/.pnpm/@nuxt+kit@3.3.1/node_modules/@nuxt/kit/dist/index.mjs:168:30)
  at async installModule (node_modules/.pnpm/@nuxt+kit@3.3.1/node_modules/@nuxt/kit/dist/index.mjs:451:15)
  at async initNuxt (node_modules/.pnpm/nuxt@3.3.1/node_modules/nuxt/dist/index.mjs:2469:7)
  at async loadNuxt (node_modules/.pnpm/nuxt@3.3.1/node_modules/nuxt/dist/index.mjs:2523:5)
  at async loadNuxt (node_modules/.pnpm/@nuxt+kit@3.3.1/node_modules/@nuxt/kit/dist/index.mjs:539:19)
  at async Object.invoke (node_modules/.pnpm/nuxi@3.3.1/node_modules/nuxi/dist/chunks/prepare.mjs:37:18)
  at async _main (node_modules/.pnpm/nuxi@3.3.1/node_modules/nuxi/dist/cli.mjs:49:20)

 ELIFECYCLE  Command failed with exit code 1.

![image](https://user-images.githubusercontent.com/177340/225049221-023130a8-dc44-4d00-873a-a7a02867258d.png)


### Additional context

_No response_

### Logs

_No response_

Same problem windows 10 node 19.4, added `"resolutions": { "@nuxt/kit": "3.2.3" }` but now I cannot perform `npm install` due to this error:

```
 npm install
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated tslint@6.1.3: TSLint has been deprecated in favor of ESLint. Please see https://github.com/palantir/tslint/issues/4534 for more information.

> postinstall
> nuxt prepare

Nuxi 3.3.1                                                                                                                                                                          12:06:35

 ERROR  The "name" argument must be specified                                                                                                                                       12:06:37  

  at new NodeError (node:internal/errors:399:5)
  at Performance.mark (node:internal/perf/performance:125:13)
  at normalizedModule (/C:/U---ject/node_modules/@nuxt/kit/dist/index.mjs:168:30)
  at async installModule (/C:/Use--/nuxtproject/node_modules/@nuxt/kit/dist/index.mjs:451:15)
  at async initNuxt (/C:/User--/nuxtproject/node_modules/nuxt/dist/index.mjs:2469:7)
  at async loadNuxt (/C:/Us--top/nuxtproject/node_modules/nuxt/dist/index.mjs:2523:5)
  at async loadNuxt (/C:/Users/stefanomartini/Desktop/nuxtproject/node_modules/@nuxt/kit/dist/index.mjs:539:19)
  at async Object.invoke (/C:/U--/nuxtproject/node_modules/nuxi/dist/chunks/prepare.mjs:37:18)
  at async _main (/C:/--p/nuxtproject/node_modules/nuxi/dist/cli.mjs:49:20)

npm ERR! code 1
npm ERR! path C:\Us---p\nuxtproject
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c nuxt prepare

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\--\AppData\Local\npm-cache\_logs\2023-03-15T11_05_23_843Z-debug-0.log
```

--- a/packages/kit/src/module/define.ts
+++ b/packages/kit/src/module/define.ts
@@ -68,9 +68,10 @@ export function defineNuxtModule<OptionsT extends ModuleOptions> (definition: Mo
     }
 
     // Call setup
-    const mark = performance.mark()
+    const key = `nuxt:module:${uniqueKey || (Math.round(Math.random() * 10000))}`
+    const mark = performance.mark(key)
     const res = await definition.setup?.call(null as any, _options, nuxt) ?? {}
-    const perf = performance.measure(`nuxt:module:${uniqueKey || (Math.round(Math.random() * 10000))}`, mark.name)
+    const perf = performance.measure(key, mark.name)
     const setupTime = Math.round((perf.duration * 100)) / 100
 
     // Measure setup time

